agents/vendor/kubeflow/tf-job/README.md

```diff
@@ -0,0 +1,76 @@
+# tf-job
```

Comment:
Should anything under `vendor/kubeflow` get checked in?

Reply:
No, now that I think of it, thank you. It could either be submoduled in or there could be a command that obtains the dependency at setup.